Gemfile

```diff
@@ -73,6 +73,8 @@ gem 'dossier'
 # Single test gem for unit testing
 gem 'single_test'
 
+gem 'daemons'
+
```

Comment:
Could you remove the duplicate gems, they cause problems in the capistrano deploy